runtime/doc/news.txt

```diff
@@ -159,7 +159,8 @@ The following new APIs and features were added.
 • |nvim_win_text_height()| computes the number of screen lines occupied
   by a range of text in a given window.
 
-• |nvim_set_keymap()| and |nvim_del_keymap()| now support abbreviations.
+• |nvim_set_keymap()|, |nvim_buf_set_keymap()|, |nvim_del_keymap()|, |nvim_get_keymap()|,
+  |nvim_buf_get_keymap()|, |nvim_buf_del_keymap()| now support abbreviations.
```

Comment:
Now I think this is a bit verbose. Maybe just this:
```suggestion
• Mapping APIs now support abbreviations.
```

Reply:
cool. done.